web/src/lightbox.js

```diff
@@ -210,6 +210,10 @@ function display_image(payload) {
     img.src = payload.source;
     $img_container.html(img).show();
 
+    $("#lightbox_overlay .image-preview > .zoom-element > img")
+        .attr("src", "")
```

Comment:
Don’t do this! The empty string is a relative URL that [refers to the current page](https://www.rfc-editor.org/rfc/rfc3986#section-4.4), and some browsers will actually attempt to reload the current page from the server, which is pretty expensive for us.

For the specific case of the `src` attribute on `<img>`, the HTML specification [prohibits an empty URL](https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-src), so even if your particular browser doesn’t have this behavior, we can’t rely on this.